src/libs/dbHandler.go

```diff
@@ -431,29 +434,59 @@ func upsertSysSummarySQL(db *sql.DB, summary types.SystemSummary, timeCount type
 	sortedLabels := strings.Split(summary.Labels, ",")
 	sort.Strings(sortedLabels)
 	summary.Labels = strings.Join(sortedLabels, ",")
+	h := sha256.New()
+	h.Write(
```

Comment:
In `common` pkg, write a function for hashing a `SystemSummary` struct and use it here.